plugins/database_api/api.cpp

```diff
@@ -122,20 +122,6 @@ namespace golos {
 
                 std::vector<account_name_type> get_miner_queue() const;
 
-                share_type get_account_reputation(const account_name_type& account) const {
-                    if (!_follow_api) {
```

Comment:
Remove all references to `follow_plugin`:
- include <golos/plugins/follow/plugin.hpp>
- the member `_follow_api` from the `plugin::api_impl`

Reply:
Done